module/VuFind/src/VuFind/Db/Service/ServiceInterface.php

```diff
@@ -22,32 +21,23 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  *
  * @category VuFind
- * @package  Db_Row
- * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @package  Database
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org Main Site
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  */
 
-namespace VuFind\Db\Row;
+namespace VuFind\Db\Service;
 
 /**
- * Row Definition for access_token
+ * Marker interface for VuFind database services.
  *
  * @category VuFind
- * @package  Db_Row
- * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @package  Database
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org Main Site
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  */
-class AccessToken extends RowGateway
+interface ServiceInterface
```

Comment:
As previously suggested, I still think we should rename this to DbServiceInterface. Also, it looks like Git made a bad decision in its rename detection logic here. :-)

Reply:
Ouch, this was an unintended and unnoticed refactoring done by my IDE.